reference/examples/CreateAccountWithAlias.md

```diff
@@ -0,0 +1,28 @@
+Create an account by setting an ED25519 or ECDSA public key alias
+
+Reference: HIP-583
+
+## Example 1
+- Create an ED25519 public key
+- Create a new account using the `AccountCreateTransaction`
+- Set the account public key alias by using `setAlias()` field and enter the ED25519 public key generated in the previous step
+- Sign with the corresponding private key of the public key generated in the first step
+- Execute the transaction
+- Show the transaction was successful by asking for the receipt and returning the transaction status
+- Get the new Hedera account ID from the receipt of the transaction
+- Get the account info using the ED25519 public key account alias in the `AccountBalanceQuery` in the `setAccountId()` field
```

Comment:
You should be able to use the account alias public key in the setAccountId() field when requesting an account info query? @Nana-EC

Reply:
So HIP 583 per writing was aimed at creating more accounts with an alias and not necessarily query them. We should consider consensus node query a separate feature request.
I do believe the mirror node already supports this though